include/clients/common/FluidNRTClientWrapper.hpp

```diff
@@ -637,7 +637,7 @@ struct StreamingControl
     for (auto outs = std::pair{outputBuffers.begin(), outputData.begin()};
          outs.first != outputBuffers.end(); ++outs.first, ++outs.second)
     {
-      if (!outs.first) continue;
+      if (!(*outs.first)) continue;
```

Comment:
This change is a bugfix for `dev`, not this PR. Please apply directly to `dev`

Reply:
ok

Reply:
done